docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -43,6 +47,8 @@ sudo apt install python2.7 python-pip git mercurial
 
 ## 2. Install Mbed CLI
 
+<span class="notes">**Note:** We suggest you install Mbed CLI from within a virtual environment and run all commands from within your virtual environment.</span>
```

Comment:
Suggest adding a brief reason: 'This will ensure that changes made by the mbed-cli installation do not propagate to the rest of your system and cause unexpected changes in behaviour in any existing Python installation'